pkg/crd/markers/crd.go

```diff
@@ -51,6 +52,9 @@ var CRDMarkers = []*definitionWithHelp{
 
 	must(markers.MakeDefinition("kubebuilder:deprecatedversion", markers.DescribesType, DeprecatedVersion{})).
 		WithHelp(DeprecatedVersion{}.Help()),
+
+	must(markers.MakeDefinition("kubebuilder:apiapprovedk8sgroup", markers.DescribesType, APIApprovedKubernetesGroup{})).
```

Comment:
Why the`k8sgroup` suffix?

Reply:
Emm, I supposed it to be short name of the annotation key `api-approved.kubernetes.io`. Do you think `k8sgroupapprovedapi` would be better?